classes/article/PublishedArticleDAO.inc.php

```diff
@@ -221,6 +224,9 @@ function getPublishedArticlesInSections($issueId, $useCache = false) {
 					$publishedArticles[$currSectionId]['title'] = $publishedArticle->getSectionTitle();
 				}
 			}
+			// get metadata of the current version
+			$publishedArticle = $this->getPublishedArticleByArticleId($publishedArticle->getId(), null, false, $publishedArticle->getCurrentVersionId());
```

Comment:
I changed `_fromRow()` to get the `latestRevisionId` if `submissionRevision==null`